example/lib/device_details/device_details_bloc.dart

```diff
@@ -77,175 +77,32 @@ class DeviceDetailsBloc {
   }
 
   void _connectTo(BleDevice bleDevice) async {
-    _bleManager.setLogLevel(LogLevel.debug);
-    var peripheral = bleDevice.peripheral;
-    peripheral
-        .observeConnectionState(emitCurrentValue: true)
-        .listen((connectionState) {
-      Fimber.d('Observerd new connection state: $connectionState');
-      _connectionStateController.add(connectionState);
-    });
-
-    Fimber.d('Try to connecto the device');
-    await peripheral.connect();
-    Fimber.d("Connected to the device");
     List<DebugLog> logs = [];
-    Function log = (text) {
+    Logger log = (text) {
       logs.insert(0, DebugLog(DateTime.now().toString(), text));
+      Fimber.d(text);
+      _logsController.add(logs);
+    };
+
+    Logger logError = (text) {
+      logs.insert(0, DebugLog(DateTime.now().toString(), text.toUpperCase()));
+      Fimber.d(text.toUpperCase());
```

Comment:
1. `Fimber.e`?
2. it is almost the same as `log` (except double `toUpperCase`) so let's just  change letters in text and call another function instead of copying the logic